### PR TITLE
PLAT-1102 | Remove display name

### PR DESCRIPTION
**Pull request by rogatty**
https://bethink.atlassian.net/browse/PLAT-1102

### CODE REVIEW

**Comment by kvas-damian** [comment]
After this change deleted accounts will be presented on frontend as: "account deleted". I think we should handle it to use Polish strings.

File: app/Models/User.php [hunk 1]
```diff
@@ -162,7 +162,13 @@ public function userProductStates() {
 
 	public function getFullNameAttribute()
 	{
-		return $this->first_name . ' ' . $this->last_name;
+		$fullName = $this->first_name . ' ' . $this->last_name;
+
+		if ($fullName === 'account deleted') {
+			$fullName = 'Konto usunięte';
```
Comment (bognix) [inline app/Models/User.php]: please use translations for this one - the API should be language agnostic.

File: app/Models/UserProfile.php [hunk 2]
```diff
@@ -48,21 +47,17 @@ public function getAvatarUrlAttribute()
 
 	public function getFullNameAttribute()
 	{
-		return "$this->first_name $this->last_name";
+		$fullName = $this->first_name . ' ' . $this->last_name;
```
Comment (bognix) [inline app/Models/UserProfile.php]: I don't like the duplication but on the other hand loading the `user` model just to get a full name is an overkill.

**Comment by rogatty** [comment]
Release on 11.04.2019

File: resources/assets/js/components/chat/PrivateChat.vue [hunk 3]
```diff
@@ -80,13 +80,13 @@ export default {
...
 	computed: {
-		...mapGetters(['isOverlayVisible', 'currentUserId', 'currentUserDisplayName']),
+		...mapGetters(['isOverlayVisible', 'currentUserId']),
 		...mapGetters('chatMessages', ['getProfileByUserId', 'profiles', 'getInterlocutor']),
 		interlocutorProfile() {
 			return this.getInterlocutor(this.room.profiles);
 		},
 		chatTitle() {
-			return this.interlocutorProfile.display_name || this.currentUserDisplayName;
+			return this.interlocutorProfile.full_name;
```
Comment (kvas-damian) [inline resources/assets/js/components/chat/PrivateChat.vue]: Do we change this fallback on purpose? Do we know if we don't use it now?

Comment (rogatty) [reply resources/assets/js/components/chat/PrivateChat.vue]: Good catch, I brought it back.

**Pull request merged**